tests/phpunit/tests/block-supports/dimensions.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @group block-supports
+ *
+ * @covers ::wp_apply_dimensions_support
+ */
+
+class WP_Block_Supports_Dimensions_Test extends WP_UnitTestCase {
+	/**
+	 * @var string|null
+	 */
+	private $test_block_name;
+
+	public function set_up() {
+		parent::set_up();
+		$this->test_block_name = null;
+	}
+
+	public function tear_down() {
+		unregister_block_type( $this->test_block_name );
+		$this->test_block_name = null;
+		parent::tear_down();
+	}
+
+	/**
+	 * Tests that minimum height block support works as expected.
+	 *
+	 * @ticket 57582
+	 *
+	 * @covers ::wp_apply_dimensions_support
+	 *
+	 * @dataProvider data_minimum_height_block_support
+	 *
+	 * @param string $block_name The test block name to register.
+	 * @param mixed $dimensions  The dimensions block support settings.
+	 * @param mixed $expected    An expected return value.
+	 */
+	public function test_minimum_height_block_support( $block_name, $dimensions, $expected ) {
+		$this->test_block_name = $block_name;
+		register_block_type(
+			$this->test_block_name,
+			array(
+				'api_version' => 2,
+				'attributes'  => array(
+					'style' => array(
+						'type' => 'object',
+					),
+				),
+				'supports'    => array(
+					'dimensions' => $dimensions,
+				),
+			)
+		);
+		$registry    = WP_Block_Type_Registry::get_instance();
+		$block_type  = $registry->get_registered( $this->test_block_name );
+		$block_attrs = array(
+			'style' => array(
+				'dimensions' => array(
+					'minHeight' => '50vh',
+				),
+			),
+		);
+
+		$actual = wp_apply_dimensions_support( $block_type, $block_attrs );
+
+		$this->assertSame( $expected, $actual );
+	}
+
+	/**
+	 * Data provider for test_minimum_height_block_support().
```

Comment:
```suggestion
	 * Data provider.
```
After discussion with other contributors yesterday: For now, data provider docblocks should just be described as "Data provider." [Context](https://github.com/WordPress/wordpress-develop/pull/3920#discussion_r1092021101)

Reply:
Good to know, thank you!